CentralComputing/UnitTests/StateMachine_Test.cpp

```diff
@@ -1,53 +1,68 @@
-#include "Sensor.h"
-#include <iostream>
-#include <unistd.h>
-#include "Pod.h"
-#include "StateMachineCompact/stdafx.h"
-#include <string>
-#include <fstream>
-#include <sstream>
-#include <algorithm>
-#include <vector>
-#include <stdlib.h>
+#include "StateMachine_Test.h"
 
-#include <time.h>
+//Include any other dependencies 
 
-using namespace std;
-
-// declaring function, implented later
-void print_help();
-void print_help_instr();
+StateMachine_Test::StateMachine_Test(){
+  name = "State Machine Test";
+  cmd  = "smt";
+}
 
-/**
-	brings out the best in all of us
-*/
-void memes() {
-	srand(time(NULL));
-	int r = rand() % 5 +1;
+int StateMachine_Test::test(int argc, char** argv) {
```

Comment:
This is a @curtislowder issue. I agree with the suggestion, but I will not be implementing it